akkeeper/src/main/scala/akkeeper/master/route/ContainerController.scala

```diff
@@ -41,24 +41,11 @@ class ContainerController(service: ActorRef)(implicit dispatcher: ExecutionConte
       path(Segment) { containerName =>
         get {
           handleRequest(service, GetContainer(containerName))
-        } ~
-        patch {
```

Comment:
I think we can also drop the following lines:
1.https://github.com/izeigerman/akkeeper/blob/8f20eafd173c564c680b957a04dda968b23c255a/akkeeper/src/main/scala/akkeeper/master/route/ContainerController.scala#L32-L34
2. https://github.com/izeigerman/akkeeper/blob/8f20eafd173c564c680b957a04dda968b23c255a/akkeeper/src/main/scala/akkeeper/master/route/ContainerController.scala#L36

Reply:
Done